templates/conftest_flask_postgres.py

```diff
@@ -4,8 +4,9 @@ def app():
     """Session-wide test `Flask` application."""
     config_override = {
         "TESTING": True,
+        # Allows for override of database to separate test from dev environments
         "SQLALCHEMY_DATABASE_URI": os.environ.get(
-            "TEST_DATABASE_URL", "postgresql://postgres:postgres@localhost:5432/postgres"
+            "TEST_DATABASE_URL", os.environ.get("DATABASE_URI")
```

Comment:
This defaults to use same database name. Generally its nicer to use a different database for tests, but it's also weird to hardcode database names like we did before. So I added a comment that this is where someone could opt for a different test database if needed. I was able to run tests and run locally afterwards so it seems to be okay for our setup.